sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/FlatMapElements.java

```diff
@@ -125,9 +129,19 @@ private FlatMapElements(
         fn, fn.getClosure(), TypeDescriptors.inputOf(fn.getClosure()), outputType);
   }
 
+  private void checkOutputType() {
+    checkState(
+        outputType != null,
+        "%s output type descriptor was null; "
+            + "this probably means that getOutputTypeDescriptor() was called after "
+            + "serialization/deserialization, but it is only available prior to "
+            + "serialization, for constructing a pipeline and inferring coders",
+        FlatMapElements.class.getSimpleName());
+  }
+
   @Override
   public PCollection<OutputT> expand(PCollection<? extends InputT> input) {
-    checkArgument(fn != null, ".via() is required");
+    checkNotNull(fn, ".via() is required");
```

Comment:
This is an additional change to keep style more consistent with the other transforms.